tools/test_runner.js

```diff
@@ -18,6 +18,8 @@ var console_wrapper = require('common_js/module/console');
 var builtin_modules =
   Object.keys(process.native_sources).concat(Object.keys(process.binding));
 
+var default_timeout = 240; // (in seconds)
```

Comment:
@kolipka great idea, I've added an additional option to `check_test.js`, please check it.